code/__DEFINES/ai.dm

```diff
@@ -7,7 +7,7 @@
 
 ///Monkey checks
 #define SHOULD_RESIST(source) (source.on_fire || source.buckled || HAS_TRAIT(source, TRAIT_RESTRAINED) || (source.pulledby && source.pulledby.grab_state > GRAB_PASSIVE))
-#define IS_DEAD_OR_INCAP(source) (HAS_TRAIT(source, TRAIT_INCAPACITATED) || HAS_TRAIT(source, TRAIT_HANDS_BLOCKED) || IS_IN_STASIS(source))
+#define IS_DEAD_OR_INCAP(source) (HAS_TRAIT(source, TRAIT_INCAPACITATED) || HAS_TRAIT(source, TRAIT_HANDS_BLOCKED) || IS_IN_STASIS(source) || source.stat)
```

Comment:
where is this used exactly?

Reply:
the ai code for monkey all over

Reply:
Should it be true if the monkey is in softcrit?

Reply:
this would do that, no?

Reply:
Yeah, just wanted to make sure it wasn't being used for "turn off" checks or anything odd, cause I'd think a monkey should be able to move in softcrit